wcfsetup/install/files/lib/system/session/SessionHandler.class.php

```diff
@@ -991,4 +984,140 @@ protected function getSpiderID($userAgent) {
 	public function isFirstVisit() {
 		return $this->firstVisit;
 	}
+	
+	/**
+	 * Returns all user sessions for a specific user.
+	 *
+	 * @return      \wcf\system\session\Session[]
+	 * @since       5.4
+	 */
+	public function getUserSessions(User $user): array {
```

Comment:
Merge this with `getAcpSessions` somehow (use a private helper method if necessary). Otherwise someone is going to forget to adjust either of these for future changes.